### DISCUSSION TITLE
Ender-3 Neo Max motherboard V4.2.7 Firmware upgrade with CR Touch.

### DISCUSSION THREAD

**Discussion by idmnstr** [CrealityOfficial/Creality_Sonic_Pad#98] [General]
I need to replace my 4.2.2 motherboard because the USB port broke off.  The V4.2.7 is available for purchase.  It requires the an update of the firmware for the Ender-3 Neo Max.  I read that I need another firmware update because of the CR Touch.  Can anyone point me to the correct firmware files I should download?

**Comment by soundgateaudio** [comment]
Yes!  The description on Amazon says I need an update for this motherboard- but when I go to the site there is no mention or selection for the Ender 3 max neo!   What the hell?  I also got the "Sprite Extruder Pro Upgrade Kit" and I can't set the Offset for the new position of the CR Touch probe!
